src/main/java/duke/command/InvalidCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Represents the InvalidCommand to deal with invalid inputs by user.
+ */
+public class InvalidCommand extends Command {
```

Comment:
Perhaps this should be an Exception as the Command class should represent the commands that the user can input?

Reply:
Alright thank you.